src/library/getIsEmptyStringOrWhitespace.js

```diff
@@ -0,0 +1,7 @@
+export const getIsEmptyStringOrWhitespace = (value) => {
```

Comment:
What if a valid value with a different type like `number` is passed in?

Reply:
It returns false because its not typeof = string. (verified with unit tests!)